File: mbrs/src/main/resources/static/js/services/TipPrikljuckaService.js
//Fri May 11 23:51:24 CEST 2018
//Generisano na osnovu sablona: jsservice.ftl
//Element modela: GremoNaElektriko::TipPrikljucka

angular.module('gremoNaElektrikoApp.TipPrikljuckaService', [])
	.factory('TipPrikljuckaService', function($http) {
		var factory = {};
		
		factory.findAll = function() {
			return $http.get('/gremoNaElektriko/tipPrikljucka');
		}
		
		factory.createTipPrikljucka = function(tipPrikljucka ) {
			return $http.post('/gremoNaElektriko/tipPrikljucka', tipPrikljucka);
		}

		factory.deleteTipPrikljucka = function(id) {
			return $http.delete('/gremoNaElektriko/tipPrikljucka/'+id);
		}
		
		factory.updateTipPrikljucka = function(tipPrikljucka) {
			return $http.put('/gremoNaElektriko/tipPrikljucka', tipPrikljucka);
		}
		
		return factory;
	});

File: mbrs/src/main/java/com/ftn/mbrs/service/TipPrikljuckaService.java
//Sat May 12 11:32:37 CEST 2018
//Generisano na osnovu sablona: service.ftl
//Element modela: GremoNaElektriko::TipPrikljucka


package com.ftn.mbrs.service;

import java.util.List;

import com.ftn.mbrs.model.TipPrikljucka;

public interface TipPrikljuckaService {

	public TipPrikljucka save(TipPrikljucka tipPrikljucka);
	
	public TipPrikljucka update(TipPrikljucka tipPrikljucka);

	public String delete(Long id);
	
	public TipPrikljucka findOne(Long id);
	
	public List<TipPrikljucka> findAll();
}
